Handle error on core level (#2167)
SAP/luigi: docs/advanced-options-luigi-client.md

  <body></body>
</html>
```

## Disable Luigi Core runtime error handling

> By default, Luigi Client listen to runtime errors and send the [error event](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent) properties to Luigi Core. 

Reviewer: ```suggestion
By default, Luigi Client listens to runtime errors and sends the [error event](https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent) properties to Luigi Core. 
```

> If a navigation node has configured a property called [onRuntimeErrorHandler](navigation-parameters-reference.md#node-parameters) you have the possibility to handle errors on core level.

Reviewer: ```suggestion
If a navigation node has configured a property called [onRuntimeErrorHandler](navigation-parameters-reference.md#node-parameters), you have the possibility to handle errors on the Luigi Core level. For example: 
```

> If a navigation node has configured a property called [onRuntimeErrorHandler](navigation-parameters-reference.md#node-parameters) you have the possibility to handle errors on core level.

Reviewer: ```suggestion
If a navigation node has configured a property called [onRuntimeErrorHandler](navigation-parameters-reference.md#node-parameters#onruntimeerrorhandler) you have the possibility to handle errors on core level.
```
